app/controllers/CalculationController.scala

```diff
@@ -16,111 +16,55 @@
-import java.time.temporal.ChronoUnit
 import config.{AppConfig, ErrorHandler}
 import controllers.predicates.AuthPredicate
 import models.User
-import models.penalty.LatePaymentPenalty
-import models.point.PointStatusEnum
-import models.v3.lpp.LPPPenaltyCategoryEnum.LPP2
-import models.v3.lpp.{LPPDetails, LPPPenaltyCategoryEnum, LPPPenaltyStatusEnum}
-import views.html.{CalculationAdditionalView, CalculationLPPView}
-
-import javax.inject.Inject
+import models.lpp.LPPPenaltyCategoryEnum.LPP2
+import models.lpp.{LPPDetails, LPPPenaltyCategoryEnum, LPPPenaltyStatusEnum}
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
 import services.PenaltiesService
-import services.v2.{PenaltiesService => PenaltiesServiceV2}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import utils.Logger.logger
 import utils.{CurrencyFormatter, EnrolmentKeys}
 import viewmodels.CalculationPageHelper
 import views.html.{CalculationAdditionalView, CalculationLPPView}
 
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
 class CalculationController @Inject()(viewLPP: CalculationLPPView,
                                       viewAdd: CalculationAdditionalView,
-                                      penaltiesService: PenaltiesService,
-                                      penaltiesServiceV2: PenaltiesServiceV2,
+                                      penaltiesServiceV2: PenaltiesService,
                                       calculationPageHelper: CalculationPageHelper)(implicit ec: ExecutionContext,
                                                                                     val appConfig: AppConfig,
                                                                                     errorHandler: ErrorHandler,
                                                                                     authorise: AuthPredicate,
                                                                                     controllerComponents: MessagesControllerComponents)
   extends FrontendController(controllerComponents) with I18nSupport with CurrencyFormatter {
 
-  def onPageLoad(penaltyId: String, isAdditional: Boolean): Action[AnyContent] = authorise.async { implicit request =>
-    logger.debug(s"[CalculationController][onPageLoad] - Making call to old endpoint")
-    getOldPenaltyData(penaltyId, isAdditional)
-  }
-
-  def onPageLoadForNewAPI(principalChargeReference: String, penaltyCategory: String): Action[AnyContent] = authorise.async { implicit request =>
-    logger.debug(s"[CalculationController][onPageLoadForNewAPI] - Making call to new endpoint")
+  def onPageLoad(principalChargeReference: String, penaltyCategory: String): Action[AnyContent] = authorise.async { implicit request =>
+    logger.debug(s"[CalculationController][onPageLoad] - Making call to new endpoint")
```

Comment:
Is this log message useful, if so, do we need to specify 'new'?

Reply:
I didn't read the actual message as you say not really useful, will remove